features/openhab-addons/src/main/feature/feature.xml

```diff
@@ -1,1055 +1,1070 @@
-<?xml version="1.0" encoding="UTF-8"?>
```

Comment:
Something went wrong here. Better replace it with the original file and add the changes again.